<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wrapper</class>
 <widget class="QMainWindow" name="wrapper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1314</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="bgImage">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>-20</y>
      <width>1531</width>
      <height>841</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>331</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding:10px 5;
background-color: rgba(251, 221, 122,0.8);

</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="show_img">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>681</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView
{
background: url(:/images/assets/images/placeholder.jpg) center;

border-radius:5px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse_img">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>680</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:#fbdd7a;
color:Black;
border-radius:5px;

	font: 75 italic 14pt &quot;Calibri&quot;;
}
QPushButton:hover
{
background:#fca22c;
color:Black;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>1. Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>395</width>
      <height>63</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit

{
background:transparent;
color:#fbdd7a;
border-radius:15px;
font: 40 Bold  30pt &quot;Calibri&quot;;
	
}
</string>
    </property>
    <property name="text">
     <string>#TAGAI - Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLogout">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>20</y>
      <width>111</width>
      <height>40</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:#ff3300;
color:#fbdd7a;
border-radius:5px;
font: 40 Bold  10pt &quot;Calibri&quot;;
}
QPushButton:hover
{
color:#fca22c;
border-color:;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>681</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 87 24pt &quot;Segoe UI Black&quot;;
color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; font-style:normal;&quot;&gt;username&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear_img">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>680</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:#fbdd7a;
color:Black;
border-radius:5px;

	font: 75 italic 14pt &quot;Calibri&quot;;
}
QPushButton:hover
{
background:#fca22c;
color:Black;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnObj_Detection">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>680</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:#fbdd7a;
color:Black;
border-radius:5px;

	font: 75 italic 14pt &quot;Calibri&quot;;
}
QPushButton:hover
{
background:#fca22c;
color:Black;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>2. Object Detection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGet_Tag">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>680</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:#fbdd7a;
color:Black;
border-radius:5px;

	font: 75 italic 14pt &quot;Calibri&quot;;
}
QPushButton:hover
{
background:#fca22c;
color:Black;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>3. Get Insta Tags</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>70</y>
      <width>207</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 0);
font: 87 10pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Detected Items in Image</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="instructions">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>551</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="instruction">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Instructions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Please check the internet before connection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Click on the browse button to select an image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Click on the objection detection to detect items in the image</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>Click on the get insta tags to get all the tags collected </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>380</y>
      <width>239</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(255, 85, 0);
font: 87 10pt &quot;Segoe UI Black&quot;;</string>
    </property>
    <property name="text">
     <string>Tags collected using Scraper</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>1172</x>
      <y>20</y>
      <width>101</width>
      <height>40</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
background:#ff3300;
color:#fbdd7a;
border-radius:5px;
font: 40 Bold  10pt &quot;Calibri&quot;;
}
QPushButton:hover
{
color:#fca22c;
border-color:;
border-radius:5px;
}</string>
    </property>
    <property name="text">
     <string>QUIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelmsg">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>500</y>
      <width>651</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>task status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>83</y>
      <width>331</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding:10px 0;
color:white;
border-radius:5px;
background-color: rgba(251, 221, 122,0.8);
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Saved Images&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QListWidget" name="items_in_img">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>100</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding:10px 5;
background-color: rgba(251, 221, 122,0.8);

</string>
    </property>
   </widget>
   <widget class="QListWidget" name="tag_in_img">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>410</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
padding:10px 5;
background-color: rgba(251, 221, 122,0.8);

</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>110</y>
      <width>211</width>
      <height>5</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;




</string>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBarTags">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>420</y>
      <width>211</width>
      <height>5</height>
     </rect>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;</string>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::BottomToTop</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="source.qrc"/>
 </resources>
 <connections/>
</ui>
